docs/src/3_Automation/04_Powers.yml

```diff
@@ -1,641 +1,740 @@
+    Tags: One or list
+
 
 Attack, Weapon:
-  Category: Combat
-  Subcategory: Weapon Attacks
-  Description: You've been trained with basic weaponry, and maybe even taken a liking to a favorite (e.g., hammer or bow). This is your go-to tool on the battlefield.
-  Mechanic: As a Major Action in combat, you make a check to attack an Enemy. This may be performed while under the effects of another Power.
-  Type: Major
-  XP Cost: 1
-  Prereq Skill:
-    - Finesse > 0
-    - Brute > 0
-  Tags:
+    Category: 
+        - Combat
+        - Weapon Attacks
+    Description: You've been trained with basic weaponry, and maybe even taken a liking to a favorite (e.g., hammer or bow). This is your go-to tool on the battlefield.
+    Mechanic: As a Major Action in combat, you make a check to attack an Enemy. This may be performed while under the effects of another Power.
+    Type: Major
+    XP: 1
+    Prereq:
+        Skill:
+            - Finesse > 0
+            - Brute > 0
 
 Attack, Charge:
-  Category: Combat
-  Subcategory: Weapon Attacks
-  Description: Your weapon training allows you throw your weight into an enemy, heightening your damage.
-  Mechanic: You must move 3 spaces before using this Power. You add +1 or 2 to the damage on a successful Weapon Attack. Power Points are still expended on a miss.
-  Type: Major
-  XP Cost: 2
-  PP Cost: [1, 2]
-  Prereq Power: Attack, Weapon
-  Prereq Role:
-    - Defender
-    - Martial
-  Tags:
+    Category: 
+        - Combat
+        - Weapon Attacks
+    Description: Your weapon training allows you throw your weight into an enemy, heightening your damage.
+    Mechanic: You must move 3 spaces before using this Power. You add +1 or 2 to the damage on a successful Weapon Attack. Power Points are still expended on a miss.
+    Type: Major
+    XP: 2
+    PP: [1, 2]
+    Prereq:
+        Power: Attack, Weapon
+    Prereq:
+        Role:
+            - Defender
+            - Martial
 
 Attack, Sweep:
-  Category: Combat
-  Subcategory: Weapon Attacks
-  Description: Your weapon training allows you throw your weight around, potentially knocking targets off balance. Sweep the legs!
-  Mechanic: You perform you Weapon Attack on multiple contiguous characters within 1 space. For 2 PP, Targets make a DR 3 Agility check vs. your TC to avoid being Knocked Down.
-  Type: Major
-  XP Cost: 2
-  PP Cost: [1,2]
-  Prereq Power: Attack, Weapon
-  Prereq Role: [Martial, Defender]
-  Tags: Multi-target
+    Category: 
+        - Combat
+        - Weapon Attacks
+    Description: Your weapon training allows you throw your weight around, potentially knocking targets off balance. Sweep the legs!
+    Mechanic: You perform you Weapon Attack on multiple contiguous characters within 1 space. 
+    Save:
+        Trigger: For 2 PP, on a hit
+        DR: 3
+        Type: AGL
+        Fail: Knocked Down
+    Type: Major
+    XP: 2
+    PP: [1,2]
+    Prereq:
+        Power: Attack, Weapon
+        Role: [Martial, Defender]
```

Comment:
They become the same thing as soon as it's imported. Do you want to adopt one or the other stylistically?

Reply:
As long as it doesn't matter that they do the exact same thing I'm good with allowing it either way. This will make the task of assimilating new ideas into this format to be less formatting specific in order to successfully render (the fewer bugs the better), so I feel like we'd be OK leaving these corrections as is. I personally will try to use the List format so we can visually see the differences better in individual sections when considering balance (it's helpful visually when I see some high level Powers don't have as many pre-reqs as I'd expect them to.)